.eslintrc.yaml

```diff
@@ -119,3 +120,6 @@ rules:
   # TODO(rye): Follow-up: DENY these
   '@typescript-eslint/explicit-module-boundary-types': warn
   '@typescript-eslint/no-empty-function': warn
+
+  # eslint-plugin-jsx-expressions – https://github.com/hpersson/eslint-plugin-jsx-expressions/
+  jsx-expressions/strict-logical-expressions: warn
```

Comment:
I agree with @hawkrives — let's make this an error today. Also, missing a newline at the end of this file

Reply:
✅ Error today 
✅ Newline at end of file